### PR TITLE
feat: e2e tests for  creating a reservation

### PR DESCRIPTION
**Pull request by dalelotts**
Also setting up the testing structure

### CODE REVIEW

**Comment by dalelotts** [comment]
I still need to verify that the environment variables are set correctly on windows

File: front-end/src/reservations/ReservationCreate.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function DishCreate() {
   }
 
   function cancelHandler() {
+    console.log("cancelled");
```
Comment (kchia) [inline front-end/src/reservations/ReservationCreate.js:28]: Should this be removed?

File: .gitignore [hunk 2]
```diff
@@ -20,7 +20,7 @@ build/
 .idea/
 .vscode/
 .vercel
-
+.screenshots/
```
Comment (kchia) [inline .gitignore:23]: Consider adding some docs on what these screenshots are and how the students can use them during the development process.

**Review by kchia — APPROVED** [review]
Looks good! Just a couple of minor comments.

**Pull request merged**